lib/common/output_text.c

```diff
@@ -328,3 +330,44 @@ pcmk__indented_printf(pcmk__output_t *out, const char *format, ...) {
     pcmk__indented_vprintf(out, format, ap);
     va_end(ap);
 }
+
+void
+pcmk__text_prompt(const char *prompt, bool echo, char **dest)
+{
+    int rc = 0;
+    struct termios settings;
+
```

Comment:
We should either assert that prompt and dest are non-null, or handle null reasonably

Reply:
I think it would make the most sense to have these required.  I don't know what the function would do otherwise besides just return, in which case just don't call it.